packages/aws-cdk/lib/cli/cdk-toolkit.ts

```diff
@@ -1218,30 +1211,23 @@ export class CdkToolkit {
   }
 
   public async refactor(options: RefactorOptions): Promise<number> {
-    if (!options.dryRun) {
-      info('Refactor is not available yet. Too see the proposed changes, use the --dry-run flag.');
+    const tk = new Toolkit();
```

Comment:
We are already setting this up in L186 and make it available as `this.toolkit` - any reason we cannot use that instance here?